docs/fundamentals/diagnostics/runtime-tiered-compilation-events.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Tiered compilation runtime events"
+description: See .NET runtime tiered compilation events that collect diagnostic information about tiered compilation in .NET Core.
+ms.date: "9/15/2023"
+ms.topic: reference
+helpviewer_keywords:
+  - "Tiered compilation events (CoreCLR)"
+  - "ETW, tiered compilation events (CoreCLR)"
+---
+# .NET runtime tiered compilation events
+
+These events collect information about tiered compilation. For more information about how to use these events for diagnostic purposes, see [logging and tracing .NET applications](../../core/diagnostics/logging-tracing.md)
```

Comment:
```suggestion
The events described in this article collect information about tiered compilation. For more information about how to use these events for diagnostic purposes, see [logging and tracing .NET applications](../../core/diagnostics/logging-tracing.md)
```

Reply:
Made a similar change in a few other files.